renku/core/management/datasets.py

```diff
@@ -1488,7 +1462,7 @@ def _check_url(url):
         if not is_git:
             # NOTE: Check if the url is a redirect.
             url = requests.head(url, allow_redirects=True).url
-            u = parse.urlparse(url)
+            _ = parse.urlparse(url)
```

Comment:
I know I wrote this, but now I'm confused, does this line actually do anything other than raise an uncaught error if the redirect url is not a valid url? I feel like we could remove this line? Looks like a copy&paste error by me